.github/ISSUE_TEMPLATE/_bug_report.md

```diff
@@ -0,0 +1,36 @@
+
+---
+name: Bug report
+about: Report a bug unrelated to any particular site or extractor
+label: check relevant labels to the issue
+---
+
+[provide general introduction to the issue logging and why it is relevant to this repository]
+
+## Context
+
+[provide more detailed introduction to the issue itself and why it is relevant]
+
+## Process
+
+[ordered list the process to finding and recreating the issue, example below]
+
+1. go to this url https://scholia.toolforge.org/author/
+2. click the random author button
+3. the error is raised
+
+## Expected result
+
+[describe what you would expect to have resulted from this process]
```

Comment:
Suggestion: add the suggestion to people to upload screenshots here too

Reply:
done